blocks/page_list/templates/itinerary/Walk.jsx

```diff
@@ -0,0 +1,35 @@
+import { dateFormatted } from './Itinerary-Utils';
+import React from 'react';
+
+
+const Walk = ({title, start, meeting, remove, key}) => {
+  return (
+    <div>
```

Comment:
Bad semantics - what about this document does being a `<div>` really describe? If it has no meaning to the reader, it's just a crufty wrapper we could do without. Is the walk a section? An item in a list? Decide what this is and stay consistent, and use as few classes as possible.